### ISSUE TITLE
Most of Skungee doesn't work at all

### ISSUE THREAD

**Issue by Govindass** [Skungee/Skungee#85]
checking if a server is running, checking if a player is online on bungee.. all of that stuff doesn't work even if based on bungee console Skungee has connected to servers properly. the only thing I found to be working is amount of bungeecord players
[12:50:17] [Server thread/ERROR]: #!#! 
[12:50:17] [Server thread/ERROR]: #!#! [Skript] Severe Error:
[12:50:17] [Server thread/ERROR]: #!#! 
[12:50:17] [Server thread/ERROR]: #!#! Something went horribly wrong with Skript.
[12:50:17] [Server thread/ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[12:50:17] [Server thread/ERROR]: #!#! You're running outdated version of Skript! Please try updating it NOW; it might fix this.
[12:50:17] [Server thread/ERROR]: #!#! You may download new version of Skript at https://github.com/bensku/Skript/releases
[12:50:17] [Server thread/ERROR]: #!#! You will be given instructions how to report this error if it persists with latest Skript.
[12:50:17] [Server thread/ERROR]: #!#! Issue tracker: https://github.com/SkriptLang/Skript/issues (only if you know what you're doing!)
[12:50:17] [Server thread/ERROR]: #!#! 
[12:50:17] [Server thread/ERROR]: #!#! Stack trace:
[12:50:17] [Server thread/ERROR]: #!#! java.lang.NullPointerException
[12:50:17] [Server thread/ERROR]: #!#!     at me.limeglass.skungee.spigot.elements.conditions.CondPlayerOnline.check(CondPlayerOnline.java:23)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Condition.run(Condition.java:56)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Conditional.walk(Conditional.java:51)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Trigger.execute(Trigger.java:55)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.command.ScriptCommand.execute2(ScriptCommand.java:285)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.command.ScriptCommand.execute(ScriptCommand.java:242)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.command.Commands.handleCommand(Commands.java:257)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.command.Commands$1.onPlayerCommand(Commands.java:172)
[12:50:17] [Server thread/ERROR]: #!#!     at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:35)
[12:50:17] [Server thread/ERROR]: #!#!     at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78)
[12:50:17] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[12:50:17] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.Skript.dispatchCommand(Skript.java:1179)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.effects.EffCommand.execute(EffCommand.java:84)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Effect.run(Effect.java:52)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Trigger.execute(Trigger.java:55)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.command.ScriptCommand.execute2(ScriptCommand.java:285)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.command.ScriptCommand.execute(ScriptCommand.java:242)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.command.Commands.handleCommand(Commands.java:257)
[12:50:17] [Server thread/ERROR]: #!#!     at ch.njol.skript.command.Commands$1.onPlayerCommand(Commands.java:172)
[12:50:17] [Server thread/ERROR]: #!#!     at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:35)
[12:50:17] [Server thread/ERROR]: #!#!     at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78)
[12:50:17] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[12:50:17] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513)
[12:50:17] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_12_R1.PlayerConnection.handleCommand(PlayerConnection.java:1484)
[12:50:17] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1297)
[12:50:17] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:45)
[12:50:17] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:5)
[12:50:17] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_12_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:14)
[12:50:17] [Server thread/ERROR]: #!#!     at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[12:50:17] [Server thread/ERROR]: #!#!     at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[12:50:17] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46)
[12:50:17] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:850)
[12:50:17] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:423)
[12:50:17] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:774)
[12:50:17] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:666)
[12:50:17] [Server thread/ERROR]: #!#!     at java.lang.Thread.run(Thread.java:748)
[12:50:17] [Server thread/ERROR]: #!#! 
[12:50:17] [Server thread/ERROR]: #!#! Version Information:
[12:50:17] [Server thread/ERROR]: #!#!   Skript: 2.2-dev36 (OUTDATED)
[12:50:17] [Server thread/ERROR]: #!#!   Bukkit: 1.12.2-R0.1-SNAPSHOT
[12:50:17] [Server thread/ERROR]: #!#!   Minecraft: 1.12.2
[12:50:17] [Server thread/ERROR]: #!#!   Java: 1.8.0_211 (Java HotSpot(TM) 64-Bit Server VM 25.211-b12)
[12:50:17] [Server thread/ERROR]: #!#!   OS: Linux amd64 3.16.0
[12:50:17] [Server thread/ERROR]: #!#! 
[12:50:17] [Server thread/ERROR]: #!#! Server platform: Paper
[12:50:17] [Server thread/ERROR]: #!#! 
[12:50:17] [Server thread/ERROR]: #!#! Current node: null
[12:50:17] [Server thread/ERROR]: #!#! Current item: [Skungee] CondPlayerOnline- Syntax: [bungee[[ ]cord] [(player|uuid)] %string/player% (1?is|2?is(n't| not)) online [the] [bungee[[ ]cord]], [(player|uuid)] %string/player% (1?is|2?is(n't| not)) online [the] bungee[[ ]cord]]
[12:50:17] [Server thread/ERROR]: #!#! Current trigger: command /tell (simple event) (love.sk, line -1)
[12:50:17] [Server thread/ERROR]: #!#! 
[12:50:17] [Server thread/ERROR]: #!#! Thread: Server thread
[12:50:17] [Server thread/ERROR]: #!#! 
[12:50:17] [Server thread/ERROR]: #!#! Language: english
[12:50:17] [Server thread/ERROR]: #!#! Link parse mode: LENIENT
[12:50:17] [Server thread/ERROR]: #!#! 
[12:50:17] [Server thread/ERROR]: #!#! End of Error.
[12:50:17] [Server thread/ERROR]: #!#!

**Comment by TheLimeGlass** [comment]
Please provide the version of Skungee you are using.

**Comment by Govindass** [comment]
> Please provide the version of Skungee you are using. Both Spigot and Bungeecord Skungee should be the same version.

they all are same version, 1.0.10

**Comment by Govindass** [comment]
I found bungeecord kicking to not work also

**Comment by Tai1er** [comment]
Hello
Same for me.
I have just updated from 1.0.8 to 1.0.10 on both bungeecord and 2 servers.
"kick %player% from bungee" and "server %string%" is running" doesn't work.
No errors.
All the servers are connected to Skungee.
Bungee and Skungee can send players to an other server.

**Comment by ham1255** [comment]
i confirm this
```
execute bungeecord command "alert hi"
```
is not working appearntly it does send the packet but Bungee aint executing it
```
[Skungee] Recieved packet: BUNGEECOMMAND with data: [alert hi] with settable data: 0
```

**Comment by ham1255** [comment]
I guess receiving part of bungeecord is broken

**Comment by ham1255** [comment]
Global script broken too, as server starts it picks latest skript file from skungee but while server is running it doesn't get it anymore like why i need to restart the server to update a skript file

**Comment by CrazyDe1999** [comment]
I guess Skungee is useless now, please author fix this none of my skripts that uses skungee works anymore

**Comment by TheLimeGlass** [comment]
Please check and make sure it is only 
"kick %player% from bungee", "server %string%" is running" and "execute bungee command %string%" exactly.

**Comment by ham1255** [comment]
Yep it doesnt work server gets the packet but it doesn't get executed

**Comment by TheLimeGlass** [comment]
I'm looking if it's only those syntax exactly. I need to know all syntax that aren't.

**Comment by CrazyDe1999** [comment]
as @ham1255 said packets sent to bungee which means part of bungeecord is not functioning correctly and these syntax are effected. and some too which means

**Comment by CrazyDe1999** [comment]
effects are not handled by bungeecord/waterfall

**Comment by ham1255** [comment]
yep those syntax are not working

**Comment by Govindass** [comment]
here's the list of all syntaxes which I found to not work:
execute bungee command
bungeecord player is online
kick player from bungee
server is running
current server (wasn't working in older version too)
bungeecord player limit (wasn't working in older version too)
bungeecord online mode (wasn't working in older version too)
bungeecord version (wasn't working in older version too)

WORKING:
all bungeecord players
skungee message
connect player to server
network variables

**Comment by TheLimeGlass** [comment]
Fixed in 1.0.10a
